apps/images/migrations/0006_alter_customrendition_file.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.2 on 2023-10-31 14:06
+
+from django.db import migrations
+import wagtail.images.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('apps_images', '0005_auto_20231025_1401'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='customrendition',
+            name='file',
+            field=wagtail.images.models.WagtailImageField(height_field='height', storage=wagtail.images.models.get_rendition_storage, upload_to=wagtail.images.models.get_rendition_upload_to, width_field='width'),
+        ),
+    ]
```

Comment:
@goapunk was that a missed one from wagtail upgrade?

Reply:
no, it isn't created on main. So must be something related to django